provers/scripts/encode-testing-results.ts

```diff
@@ -0,0 +1,47 @@
+#! ./node_modules/.bin/ts-node
+
+import 'zx/globals';
+import { StandardMerkleTree } from '@openzeppelin/merkle-tree';
+import * as tf from '@tensorflow/tfjs-node';
+
+import dataDriver from "../utils/load-mnist-data";
+
+const MODEL: string = argv.MODEL || "demo-tfjs";
+// const TESTING_DATA: string = argv.TESTING_DATA || "demo";
```

Comment:
Should be able to load testing data from local files, instead of fetching from some APIs for the sake of universality.

Reply:
This might be out of scope rn.